src/gridstack.ts

```diff
@@ -226,8 +221,8 @@ export class GridStack {
   protected _ignoreLayoutsNodeChange: boolean;
   /** @internal */
   public _gsEventHandler = {};
-  /** @internal */
-  protected _styles: GridCSSStyleSheet;
+  /** @internal max height of the gridstack*/
+  public _max: number;

```

Comment:
does it need to be public ? if so '_' isn't the right prefix, and rename to curRow or something (need to see how it's used)